cdap-cli/src/main/java/io/cdap/cdap/cli/CLIConfig.java

```diff
@@ -183,7 +183,7 @@ private void checkConnection(ClientConfig baseClientConfig, ConnectionConfig con
     try {
       metaClient.ping();
     } catch (IOException e) {
-      throw new IOException("Check hostname and/or port", e);
+      throw new IOException("Check hostname and/or port and/or accesstoken", e);
```

Comment:
A better error message? May be something like "Check connection parameters and/or accesstoken". Would invalid accesstoken throw IOException? Or it will be for invalid access token file?

Reply:
Depending on whether it is sandbox or not different response comes back from the server which is caught in 'MetaClient' and thrown as IO exception. Will change the error message.

Reply:
Done.